### ISSUE TITLE
Model our database

### ISSUE THREAD

**Issue by Vernalhav** [flossschool/pollex#24]
We must define which fields are going to be present in our database.

**Comment by Vernalhav** [comment]
I think we should have two tables: one for users and one for polls. Each user and each poll should have a unique ID so that they can be referenced.
The user should have a list of poll IDs they have created (and maybe another list with the IDs they have participated in).
The poll should have the ID of the user that created it and how many votes there are for each category. I believe adding in more information is not our priority since we aim to release an MVP.

**Comment by leodaher** [comment]
I agree with @Vernalhav , two tables, one for the users and the other for polls, is fine. What database system should we use? What do you think of PostgreSQL, so we can use SQLAlchemy, which is a widely used ORM. There's even Flask-SQLAlchemy, which is specific for Flask.

**Comment by leodaher** [comment]
I've made a PR to configure Flask-SQLAlchemy, and the database. https://github.com/flossschool/pollex/pull/33

**Comment by gdezan** [comment]
I've created a pull request to keep working on @leodaher 's changes https://github.com/flossschool/pollex/pull/35

**Comment by LuisF3** [comment]
@ET33 was working on it last class. Have you finished it?

**Comment by leodaher** [comment]
I guess with @gdezan PR, it is done.

**Comment by LuisF3** [comment]
I left a comment there (PR #35 ). I think we need a bit more modeling and there I explained some flaws I could find. Let me know what you guys think.

And please, leave a reference when referencing a PR or Issue :)

**Comment by ET33** [comment]
Thanks for starting this. I think only 2 tables is insufficient to test the voting system, etc. So I've suggested some other tables [here](https://github.com/flossschool/pollex/pull/35#issuecomment-565103770)
